testing/compatibility/azure/deploy.ps1

```diff
@@ -10,76 +10,85 @@ $env:adminPassword = $adminPassword
 
 $env:octopusServerRole = 'OsTestingListening'
 $env:octopusServerEnvironment = 'OsTesting'
-$env:tentacleVersion = '7.0.201'
+$env:tentacleVersion = '8.1.32'
 
+## Takes this on to the end of the tentacle's name as seen in octopus
+$env:tentacleNamePostfix = "$env:tentacleVersion"
```

Comment:
This means its really easy to see what the tentacle is from the octopus side.

Reply:
Nit: I don't think `postfix` is a real word. Suffix?